src/pages/Home.jsx

```diff
@@ -1,19 +1,23 @@
 import Navbar from "@/components/homepage/navbar/Navbar";
-import DocFileIcon from "@/assets/home_doc_file_icon.svg";
 import DocSortingIcon from "@/assets/home_doc_sorting_icon.svg";
 import DocApi from "@/util/document_apis";
-import Moment from 'moment';
-import { Link } from "react-router-dom";
+import GridSelector from "@/components/homepage/doc_selectors/GridSelector"
 
 import DocSortingExpandedIcon from "@/assets/home_doc_sorting_expand_icon.svg";
 import DocSortingExpandIcon from "@/assets/expand_right.svg"
 import Popover from '@mui/material/Popover';
 import { useReducer, useState } from "react";
 import Clickable from "@/components/common/Clickable";
-import { SORTING_OPTIONS } from "../util/constants";
+import { HOMEPAGE_DOC_LAYOUTS, SORTING_OPTIONS } from "../util/constants";
```

Comment:
Ok thought about this, what if we kept the Enums in the files where we use them? because those enums (iirc) are only used there

Reply:
Hmmm ok yes that's a very fair point, I'm down to keep the constants in their respective files unless we need them elsewhere in which case we can move them over to the constants file.